src/main/java/com/redhat/red/build/koji/KojiClientUtils.java

```diff
@@ -39,9 +39,14 @@ public static MultiCallRequest buildMultiCallRequest( String method, List<Object
             {
                 if ( registry.hasRender( arg.getClass() ) )
                 {
-                    arg = registry.renderTo( arg );
+
+                    Object obj = registry.renderTo( arg );
+                    builder.addCallObj( method, obj );
+                }
+                else
+                {
+                    builder.addCallObj( method, arg );
```

Comment:
Can this happen? Is it better to throw an exception if the object cannot be rendered?

Reply:
Yeah. This can happen when the parameters are primitives.